bundles/org.openhab.core/src/main/java/org/openhab/core/items/HistoricStateChangeListener.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.items;
+
+import java.time.ZonedDateTime;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.types.State;
+
+/**
+ * <p>
+ * This interface must be implemented by all classes that want to be notified about changes in the state of an item.
+ *
+ * <p>
+ * The {@link GenericItem} class provides the possibility to register such listeners.
+ *
+ * @author Kai Kreuzer - Initial contribution
+ */
+@NonNullByDefault
+public interface HistoricStateChangeListener extends StateChangeListener {
```

Comment:
I would prefer the new method to be integrated in the old `StateChangeListener` interface with a default implenentation that throws an `UnsupportedOperationException`.